pkgs/applications/audio/bschaffl/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, pkg-config, cairo, libX11, lv2 }:
+
+stdenv.mkDerivation rec {
+  pname = "bschaffl";
```

Comment:
The name should be lowercase: https://nixos.org/nixpkgs/manual/#sec-package-naming . (Indeed it is inconvenient when names differ.)